Gruntfile.js

```diff
@@ -100,7 +100,7 @@ module.exports = function(grunt) {
   // load tasks
   grunt.loadNpmTasks('grunt-contrib-clean');
   grunt.loadNpmTasks('grunt-contrib-copy');
-  grunt.loadNpmTasks('grunt-replace');
+  grunt.loadNpmTasks('grunt-replace-regex');
```

Comment:
this is the same repo, just someone forked it so they could update the dependencies

Reply:
ok cool, looks like the forker seems to be a lead at Esri, seems legit, we don't have any bitcoin to steal anyway so this should be fine 😛